api/src/main/resources/metadata/Concepts_Programs-1.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<dataset>
+    <concept_class concept_class_id="11" name="Misc" description="Terms which don't fit other categories" creator="1" date_created="2004-03-02 00:00:00.0" retired="false" retired_by="1" uuid="8d492774-c2cc-11de-8d13-0010c6dffd0f"/>
+    <concept_class concept_class_id="18" name="Program" description="Program concept" creator="1" date_created="2011-01-18 01:53:56.0" retired="false" retired_by="1" uuid="1edca368-768a-102f-83f4-12313b04a615"/>
```

Comment:
Removal of this line was the cause of concepts not being imported. That class is missing in UgandaEMR and therefore it was lacking correct mapping to HIV Program. I just reintroduced it.